api/v1beta1/tempest_types.go

```diff
@@ -62,6 +63,15 @@ type TempestSpec struct {
 	// Tempest Container Image URL (will be set to environmental default if empty)
 	ContainerImage string `json:"containerImage"`
 
+	// +kubebuilder:validation:Optional
+	// persistentVolumeClaimVolumeSource represents a reference to a
+	// PersistentVolumeClaim in the same namespace. Usable for storing test results.
+        PersistentVolumeClaim *corev1.PersistentVolumeClaimVolumeSource `json:"persistentVolumeClaim,omitempty"`
```

Comment:
This is just an idea. I know that we might be changing this later but don't we want to choose more descriptive name? Something like testResultsPV?